packages/types/src/Metadata/latest.json

```diff
@@ -1473,7 +1473,7 @@
             "name": "VoteOf",
             "modifier": 0,
             "type": {
-              "key": "(ReferendumIndex, AccountId)",
+              "key": "( ReferendumIndex , AccountId )",
```

Comment:
This one is weird - I'm assuming it relates to the same issue, but basically above we have all spaces in tuples whacked, here they are added. (Thank heavens we have trims in createType)